Thanks @varmenise 

I have checked the tests and none seems to be related:

- Regression - hudson.model.ViewPropertyTest.testInvisibleProperty --> java.lang.OutOfMemoryError: GC overhead limit exceeded
- Regression - hudson.model.ViewPropertyTest.testRoundtrip --> java.lang.OutOfMemoryError: GC overhead limit exceeded
- Regression - hudson.model.UserTest.testDoConfigSubmit --> Caused by: java.lang.OutOfMemoryError: Java heap space
- Regression - hudson.model.UserTest.testDoDoDelete --> Caused by: java.lang.OutOfMemoryError: Java heap space
- Regression - hudson.bugs.seasar.Operation2174Test.testBuildChains  --> java.net.SocketTimeoutException: Read timed out

--- a/core/src/main/java/hudson/util/ArgumentListBuilder.java
+++ b/core/src/main/java/hudson/util/ArgumentListBuilder.java
@@ -342,7 +342,7 @@ public class ArgumentListBuilder implements Serializable, Cloneable {
                 percent = (c == '%');
                 if (quoted) quotedArgs.append(c);
             }
-            if(i == 0) quotedArgs.append('"');
+            if(i == 0 && quoted) quotedArgs.insert(0, '"'); else if (i == 0 && !quoted) quotedArgs.append('"');
             if (quoted) quotedArgs.append('"'); else quotedArgs.append(arg);
             
             windowsCommand.add(quotedArgs, mask.get(i));

